README.md

```diff
@@ -1,6 +1,6 @@
 # Splunk logging for .NET 
 
-#### Version 1.7.1
+#### Version 1.7.2
```

Comment:
Let's roll back all version number changes since we'll be doing that in another PR... probably as v2

Reply:
@shakeelmohamed Can we not release this PR to support `.Net Core`? I have created a custom build using this PR. I have been using it for a while now in `.Net Core` Project. It is quite stable. But due to my custom build, it is harder for me to update to your latest version.